.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: node_js
 node_js:
-- '6.4'
+- '8.9'
 env:
 - CXX=g++-4.8
+services:
+  - elasticsearch
+before_script:
+  - sleep 10
```

Comment:
I don't suppose there's any way to put a comment in a `.travis.yml` file. I hate these little bits of "I had to do this to get things to work" cruft – it's so hard to leave breadcrumbs so that later someone can figure out why the hack was needed. /rant